Playbooks/AS-AI-Commandline-Analysis/azuredeploy.json

```diff
@@ -0,0 +1,288 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "Comments":  "This playbook is intended to be run from a Microsoft Sentinel incident that has a Commandline entity. It will send a prompt to GPT-3 (text-davinci-003 model) and return an explanation of the commandline to the comments of the incident.",
+        "Author": "Curtis Middlehurst"
+    },
+    "parameters": {
+        "playbook_name": {
+            "defaultValue": "",
+            "type": "string"
+        },
+        "userprincipalname":{
+            "defaultValue": "user@domain",
+            "type": "string"
```

Comment:
1. Can you use PascalCase for parameter names? Like PlaybookName instead of  playbook_name. It renders better. 
If you update the parameter name, don't forget to update wherever it is being used. 
2. You can provide default value for playbook name parameter as well.

Reply:
Thanks, pascal case issue resolved in 40730a1.

The 'userprincipalname' parameter has also been removed as managed identity is now used for this connection.